### ISSUE TITLE
count param to schedule?

### ISSUE THREAD

**Issue by ComLock** [enonic/lib-cron#1]
If you want to run a callback only once, or perhaps twice, it would be nice to have a parameter to do that.

I terms of running a callback once 5 minutes from now, or running some callback when a condition occurs (like some long running task is complete) I guess the event API or some other API would be better suited than lib-cron.
